conbench/templates/user-list.html

```diff
@@ -63,7 +63,7 @@
     // published in https://cdn.datatables.net/plug-ins/1.13.3/features/.
     // kudos to https://stackoverflow.com/a/29639664/145400
     "conditionalPaging": true,
-    "pageLenth": 50,
+    "pageLength": 50,
```

Comment:
the outfall was: the table showed 10 entries. So, not too bad. But yeah -- linting would help. Rigorous manual testing is almost impossible.